lib/middleware.py

```diff
@@ -302,16 +302,16 @@ def post_rbac_workspace(name, parent_id, description) -> UUID | None:
     return UUID(resp_data["id"])
 
 
-def rbac_create_ungrouped_hosts_workspace(identity: Identity) -> UUID | None:  # noqa: ARG001, used later
+def rbac_create_ungrouped_hosts_workspace() -> UUID | None:  # noqa: ARG001, used later
```

Comment:
I had assumed that we would need the Identity for this call, but I was wrong - we just pass the header through. Since we don't actually need it, we can remove the noqa.
```suggestion
def rbac_create_ungrouped_hosts_workspace() -> UUID | None: 
```